docs/config-description.md

```diff
@@ -1,452 +1,600 @@
+- **Description**: The maximum size of the log directory in megabytes.
+
+### log_rotation_hour_interval
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 12
-- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`
- -   **Description**: Interval in hours for log rotation. If the current log file reaches this value in logging, a new log file starts.
-### Key: log_tag_style
+- **Default value**: `12`
+- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`.
+- **Description**: Represents the interval (in hours) for log rotation. If the current log file reaches this value in logging, a new log file starts.
+
+### log_tag_style
+
 - **Required**: True
 - **Type**: string
-- **Default value**: none
-- **Constraints**: The value must be one of the following: `int, uint, null, none, uuid`
- -   **Description**: Style for log tags.
-### Key: extractor_threads
+- **Default value**: `none`
+- **Constraints**: The value must be one of the following: `int`, `uint`, `null`, `none`, `uuid`.
+- **Description**: The style for log tags.
+
+### extractor_threads
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Number of extractor threads.
-### Key: read_only
+- **Default value**: `1`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of extractor threads.
+
+### read_only
+
 - **Required**: True
 - **Type**: boolean
-- **Default value**: True
+- **Default value**: `True`
 - **Constraints**: None
- -   **Description**: Indicates if the server should have read-only privileges.
-### Key: txn_threshold
+- **Description**: Indicates if the server should have read-only privileges.
+
+### txn_threshold
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 0
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Transaction threshold value.
-### Key: start_sequence
+- **Default value**: `0`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: The transaction threshold value.
+
+### start_sequence
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Starting ledger index.
-### Key: finish_sequence
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: Starting ledger index.
+
+### finish_sequence
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Ending ledger index.
-### Key: ssl_cert_file
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The ending ledger index.
```

Comment:
Similarly to above, this needs more detail. Like "Clio will stop writing ledger data to the DB once the specified ledger index is ingested"